mm2src/mm2_main/src/lp_ordermatch.rs

```diff
@@ -3927,19 +3819,20 @@ pub async fn lp_auto_buy(
         order.p2p_keypair(),
     );
 
-    let result = json!({ "result": LpautobuyResult {
+    let res = try_s!(serde_json::to_vec(&Mm2RpcResult::new(SellBuyResponse {
```

Comment:
`let res = try_s!(json::to_vec(&Mm2RpcResult::new(SellBuyResponse {` since `serde_json` is imported as `json`

Reply:
[done](https://github.com/KomodoPlatform/atomicDEX-API/pull/1768/commits/107c054490f6282e884ec6299ff49db91e6281ea)